GeoLib/GEOObjects.h

```diff
@@ -339,8 +339,9 @@ class GEOObjects final
      * @param pnt_offsets offsets in the merged vector storing the points
      * @return true, if merging the points succeeded, else false
      */
-    bool mergePoints(std::vector<std::string> const & geo_names, std::string & merged_geo_name,
-            std::vector<std::size_t> &pnt_offsets);
+    bool mergePoints(std::vector<std::string> const& geo_names,
+                     std::string& merged_geo_name,
```

Comment:
in mergeSurfaces and polylines, this is const&. But could be modifying, don't know.
```suggestion
                     std::string const& merged_geo_name,
```

Reply:
`mergePoints` calls `addPointVec` which is allowed to modify the string.